app/services/album.js

```diff
@@ -0,0 +1,34 @@
+const fetch = require('node-fetch');
+const { url } = require('../../config').common.api;
+const errors = require('../errors');
+
+exports.getAlbum = async id => {
+  const endpoint = `${url}albums/${id}`;
+  try {
+    const album = await fetch(endpoint);
```

Comment:
why did you use await instead promises?

Reply:
I use to put await to avoid unresolved promises, but I can change it

Reply:
We tend to use the functional programming paradigm, so in that context promises are less imperative than async await. That's the only reason i'll change this